src/providers/https.ts

```diff
@@ -204,6 +204,8 @@ export class HttpsError extends Error {
 
   /**
    * A wire format representation of a provided error code.
+   *
+   * @hidden
```

Comment:
Let's move @hidden to the top of the comment to be consistent with the rest.

Reply:
`@hidden` is not used in combination with actual documentation anywhere in the codebase yet. When you hover over the use of it in any IDE, the thing you see is the content of JSDoc, that's why I'd prefer to have human-readable things above machine-readable. WDYT?

Reply:
Sure, I don't have strong feelings here, if it makes it easier to see in the IDE then let's keep it as is.